it/stubs/AuthStub.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package stubs
+
+import com.github.tomakehurst.wiremock.stubbing.StubMapping
+import play.api.http.Status._
+import play.api.libs.json.{JsObject, Json}
+import support.WireMockMethods
+
+object AuthStub extends WireMockMethods {
+
+  private val authoriseUri: String = "/auth/authorise"
+
+  private val enrolment: JsObject = Json.obj(
+    "key"         -> "HMRC-TERS-ORG",
+    "identifiers" -> Json.arr(
+      Json.obj(
+        "key"   -> "SAUTR",
+        "value" -> "7000040245"
+      )
+    )
+  )
```

Comment:
⛏️ Don't we have writes to construct this in a less error-prone way?

Reply:
The credentials model lacks JSON writes to utilise, but the enrolment model does have one so I have replaced this value with an implementation using the appropriate case class, and used the appropriate writes to populate the JSON